packages/markers/src/browser/problem/problem-frontend-module.ts

```diff
@@ -34,4 +36,6 @@ export default new ContainerModule(bind => {
             ctx.container.get(ProblemContribution)
         ).inSingletonScope();
     }
+    bind(ProblemDecorator).toSelf().inSingletonScope();
+    bind(TreeDecorator).toDynamicValue(ctx => ctx.container.get(ProblemDecorator)).inSingletonScope();
```

Comment:
I think we should use different symbols for different trees, here it should be `NavigatorTreeDecorator` symbol. Otherwise, `ProblemDecorator` will be used for all trees.